hipcub/include/hipcub/backend/rocprim/device/device_spmv.hpp

```diff
@@ -139,7 +139,7 @@ template <typename ValueT>
         }
         else
         {
-            size_t block_size = min(num_cols, DeviceSpmv::CsrMVKernel_MaxThreads);
+            size_t block_size = min(static_cast<int>(num_cols), DeviceSpmv::CsrMVKernel_MaxThreads);
```

Comment:
the fix does not work.

should be

size_t block_size = min(num_cols, static_cast<int>(DeviceSpmv::CsrMVKernel_MaxThreads));